src/System.Runtime.Serialization.Formatters/pkg/System.Runtime.Serialization.Formatters.pkgproj

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>  
+    <ProjectReference Include="..\ref\System.Runtime.Serialization.Formatters.csproj">
+      <SupportedFramework>net462;netcoreapp1.0</SupportedFramework>
```

Comment:
Why only here?  Can we OOB this back to net46/netcore50/netstandard1.3 at least?